iac-extensions/cloudformation/src/main/java/org/sonar/iac/cloudformation/plugin/CloudformationSettings.java

```diff
@@ -70,7 +70,7 @@ public static List<PropertyDefinition> getExternalReportProperties() {
     return List.of(PropertyDefinition.builder(CFN_LINT_REPORTS_KEY)
       .index(33)
       .name("Cfn-Lint Report Files")
-      .description("Paths (absolute or relative) to the files with Cfn-Lint issues.")
+      .description("Paths (absolute or relative) to the files with Cfn-Lint issues. You can use Ant patterns to define paths.")
```

Comment:
I would suggest to call it ` WildcardPatterns` as it's not completely ant but more ant style and users can find  WildcardPattern in our documentation.